src/pfe/file-watcher/server/test/cronjob/cronjob-win.sh

```diff
@@ -95,16 +97,33 @@ function checkExitCode() {
 	fi
 }
 
+function createProject() {
+   ./$EXECUTABLE_NAME project create --url $1 $2
+}
+
+function copyToPFE() {
+   docker cp $1 $CW_CONTAINER:"/codewind-workspace"
+}
+
+echo -e "${BLUE}>> Downloading latest installer ... ${RESET}"
+EXECUTABLE_NAME="cwctl"
+curl -X GET http://download.eclipse.org/codewind/codewind-installer/master/latest/cwctl-win.exe --output $EXECUTABLE_NAME
```

Comment:
Since we're doing a curl output to current location, will the next cron run overwrite the old executables?

Reply:
we remove it after the test run